packages/terra-application-utility/tests/wdio/header-utility-menu-spec.js

```diff
@@ -1,19 +1,13 @@
 Terra.describeViewports('HeaderUtilityMenu', ['huge'], () => {
-  before(() => {
-    browser.url('/#/raw/tests/terra-application-utility/application-utility/default-header-utility-menu');
+  it('Displays a default header utility menu', () => {
+    browser.url('/raw/tests/terra-application-utility/application-utility/default-header-utility-menu');
     browser.waitForVisible('#default');
+    Terra.validates.element('default', { selector: '#default' });
   });
 
-  describe('Displays a default header utility menu', () => {
-    Terra.it.validatesElement({ selector: '#default' });
-  });
-
-  describe('Focus-header utility menu', () => {
-    before(() => {
-      browser.keys(['Tab', 'Tab']);
-      browser.moveToObject('#root', 0, 0);
-    });
-
-    Terra.it.validatesElement({ selector: '#default' });
+  it('Focus-header utility menu', () => {
```

Comment:
`it` blocks aren't really a replacement for `describe` blocks. `it` blocks should describe the action or assertion the block is testing.

```js
it('should focus the utility menu', () => {
  // Steps and assertions.
});
```

Reply:
Done 39e5dc2